app/src/main/res/values/strings.xml

```diff
@@ -141,29 +141,29 @@
     <string name="spec_bluetooth">Bluetooth</string>
     <string name="spec_data">Data Usage and Policy</string>
     <string name="spec_wallpaper">Wallpaper</string>
-    <string name="spec_wifiAccessPoints">Wifi Access Points</string>
+    <string name="spec_wifiAccessPoints">Wi-Fi Access Points</string>
 
-    <string name="error_creating_notification_channel">Unable to create notification channel</string>
-    <string name="error_updating_schedule_mode">Error updating %1$s of schedule %2$d</string>
-    <string name="permission_not_granted">Permission to access external storage is required</string>
-    <string name="restart_dialog">Application seems to need to be restarted</string>
+    <string name="error_creating_notification_channel">Could not create notification channel</string>
+    <string name="error_updating_schedule_mode">Could not update %1$s of the %2$d schedule</string>
+    <string name="permission_not_granted">Grant permission to access external storage first</string>
+    <string name="restart_dialog">The app needs to be restarted</string>
 
-    <string name="dialogOK">Ok</string>
+    <string name="dialogOK">OK</string>
     <string name="dialogCancel">Cancel</string>
     <string name="dialogYes">Yes</string>
     <string name="dialogNo">No</string>
     <string name="dialogSave">Save</string>
-    <string name="dialog_refuse">Won\'t do it</string>
-    <string name="dialog_approve">Let\'s do it</string>
-    <string name="dialog_start">Let\'s go</string>
+    <string name="dialog_refuse">Sounds like a bad idea</string>
+    <string name="dialog_approve">Yes</string>
+    <string name="dialog_start">Start</string>
```

Comment:
This string in particular fails in translation, as "Let's go" pretty much only has this meaning in English.